docs/topics/installing-cli-tool.adoc

```diff
@@ -44,7 +44,7 @@ When you encounter `<{ProductShortName}_HOME>` in this guide, replace it with th
 
 . Copy or save the `mta-cli` binary to a known directory.
 +
-Continue to xref:cli-run[Running the CLI] to run the CLI command.
+Continue to xref:mata-cli-run.adoc#cli-run[Running the CLI] to run the CLI command.
```

Comment:
This is still wrong. 

See the following build error:

```
Unknown ID or title "cli-run", used as an internal cross reference
/docs/Documents/SublimeMerge/windup-documentation/docs/cli-guide/build/en-US/master.xml fails to validate
```

Reply:
It works.
Thanks Andy